workspace/popjava/src/popjava/dataswaper/POPByte.java

```diff
@@ -0,0 +1,76 @@
+package popjava.dataswaper;
+
+import popjava.buffer.POPBuffer;
+/**
+ * Primitive settable byte for POPJava, needed for {@link popjava.annotation.POPParameter} direction use in methods 
```

Comment:
Oh, I just assumed they didn't work as OUT/INOUT since they were primitive. I needed an INOUT float for the fitness so I just added all primitive.